### PR TITLE
Add an ERDDAP interpolate notebook

### PR DESCRIPTION
**Pull request by ocefpaf**
Closes https://github.com/ioos/ioos_code_lab/issues/23

This notebook fetches the `sd1031_hurricane_2021` hurricane temperature data and then uses ERDDAP's interpolate functionality to fetch a track of interpolated temperatures that can be used to compare/calibrate the sensor data in the saildrone.

### CODE REVIEW

**Comment by MathewBiddle** [comment]
Just a thought. Could we use data from sd 1045? from this viral video https://www.saildrone.com/news/saildrone-collects-live-video-inside-hurricane-sam?

File: jupyterbook/content/code_gallery/data_analysis_and_visualization_notebooks/2021-10-25-ERDDAP-interpolate.ipynb [hunk 1]
```diff
@@ -0,0 +1,302 @@
+{
```
[the remark this answers is not in the document]
Comment (MathewBiddle) [reply jupyterbook/content/code_gallery/data_analysis_and_visualization_notebooks/2021-10-25-ERDDAP-interpolate.ipynb:1]: Lets continue with the saildrone data. We can make another ATN notebook when that service is available.

[the remark this answers is not in the document]
Comment (MathewBiddle) [reply jupyterbook/content/code_gallery/data_analysis_and_visualization_notebooks/2021-10-25-ERDDAP-interpolate.ipynb:1]: sounds like a good approach. As you say, they can always go back to the docs.

File: jupyterbook/content/code_gallery/data_analysis_and_visualization_notebooks/2021-10-25-ERDDAP-interpolate.ipynb [hunk 2]
```diff
@@ -0,0 +1,376 @@
...
```
[the remark this answers is not in the document]
Comment (ocefpaf) [reply jupyterbook/content/code_gallery/data_analysis_and_visualization_notebooks/2021-10-25-ERDDAP-interpolate.ipynb:1]: I'll try to implement that asap. Yeah, the current plot is quite confusing.

**Comment by ocefpaf** [comment]
@MathewBiddle that comparison was indeed confusing and I decided to leave just the line fit and the diff plot. Now the scatter plots should the track and the temperature without the using the size as a third dimension.

**Comment by MathewBiddle** [comment]
That looks better. Thanks @ocefpaf

**Comment by ocefpaf** [comment]
OK. Merging this and fixing the failing notebooks in another PR. Thanks!

**Pull request merged**